assets/js/components/DetailsPermaLinks.js

```diff
@@ -44,9 +45,10 @@ export default function DetailsPermaLinks( { title, path } ) {
 	return (
 		<Fragment>
 			<Link
-				className="googlesitekit-display-block"
-				href={ detailsURL }
+				className={ serviceURL === null ? 'googlesitekit-display-block' : '' }
```

Comment:
Even though we only have one class here, the fact that it's conditional makes it a bit cleaner and more consistent to use `classnames` instead. Also, we don't really need to check if `serviceURL === null` specifically, we can simply use `serviceURL` in a looser truthy comparison. In this case, the class should be added if `! serviceURL`.

Reply:
👍

Reply:
Sorry @Hazlitte, what I meant was that `classnames` is designed to handle conditional classes and you'll see it used this way throughout much of the codebase. [See here for more information](https://github.com/JedWatson/classnames#usage).